modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/view_form_instance_records.jsp

```diff
@@ -90,7 +90,7 @@ renderResponse.setTitle(LanguageUtil.get(request, "form-entries"));
 
 					<liferay-ui:search-container-column-text
 						name="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnName(ddmFormField) %>"
-						value="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(ddmFormField, ddmFormFieldValuesMap.get(ddmFormField.getName())) %>"
+						value="<%= ddmFormViewFormInstanceRecordsDisplayContext.getColumnValue(formInstanceRecord, ddmFormField, ddmFormFieldValuesMap.get(ddmFormField.getName())) %>"
```

Comment:
hey @linolaoj , can you please use ddmFormValues to retrieve ddmForm instead? Thanks!